skytap/resource_skytap_vm.go

```diff
@@ -422,13 +439,29 @@ func resourceSkytapVMRead(d *schema.ResourceData, meta interface{}) error {
 	if err != nil {
 		return err
 	}
+	err = d.Set("max_cpus", vm.Hardware.MaxCPUs)
+	if err != nil {
+		return err
+	}
+	err = d.Set("max_ram", vm.Hardware.MaxRAM)
+	if err != nil {
+		return err
+	}
 	if len(vm.Interfaces) > 0 {
 		if err := d.Set("network_interface", flattenNetworkInterfaces(vm.Interfaces)); err != nil {
 			log.Printf("[ERROR] error flattening network interfaces: %v", err)
 			return err
 		}
 	}
-	if vm.Hardware != nil && len(vm.Hardware.Disks) > 0 {
+
+	if len(vm.Hardware.Disks) > 0 {
```

Comment:
Is not this always true?

Reply:
Well - it is actually possible to remove all disks from a vm...